setup.cfg

```diff
@@ -3,3 +3,8 @@ python-tag = py3
 
 [aliases]
 test = pytest
+
+[tool:pytest]
+# Only run benchmarks as tests.
+# To actually run the benchmarks, use --benchmark-enable on the command line.
+addopts = --benchmark-disable
```

Comment:
We could also use benchmark-skip to completely skip them, but I agree this is better. They are probably a bit redundant, but not too slow if they run only as test.